pyproject.toml

```diff
@@ -11,7 +11,8 @@ description-file = "README.md"
 classifiers = ["License :: OSI Approved :: MIT License"]
 requires-python = ">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
 requires = [
-  "pathlib2; python_version < '3.4'"
+  "importlib-resources; python_version < '3.7'",
```

Comment:
Sorry @gaborbernat I’m adding even more dependencies 😛

Reply:
I thought some more, but this one is pretty difficult. There are currently three ways to do this: `importlib.resources`, `pkg_resources`, and `pkgutil`, all with different advantages and downsides. Maybe it would be best if the user can tell the installer what to use, but I’m not quite sure if the complexity is worth it. This is yet another thing to be considered after features are in place.

Reply:
I'm happy with ``importlib-resources`` for now 👍

Reply:
I will say I'm working towards deprecating `pkgutil` someday.